src/compat/libc/stdlib/mblen.c

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file
+ * @brief mblen function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+int mblen(const char *str, size_t max) {
+	int n = 5;
+	unsigned int ucs = *str & 0xFF;
```

Comment:
`str` pointer is used here before check for NULL-ptr.

Reply:
fixed